### PR TITLE
Added profile info component

### LINKED ISSUE #33: [Frontend] Create the profile info component
# Description

### Refer the ui design and create the profile info component

# Screenshot

image

-----
Profile page

# Colours and Information

image

# Illustration used

[untitled folder.zip](https://github.com/codeforcauseorg/causefolio/files/6124777/untitled.folder.zip)

# link

For reference you can see the design prototype : 
https://www.figma.com/proto/dMw9GAeJja2baVhDoceAEB/Untitled?node-id=9%3A48&viewport=-1544%2C148%2C0.4358828365802765&scaling=contain

### PR DESCRIPTION
**Pull request by nishant-rajput**
### Description

Created profile info component inside `src/components` with the name `ProfileInfo.js`

### Related Issue

Fixes #33 

### Type of Change:

- User Interface

### How Has This Been Tested?

Tested on local node environment.

### Checklist:

- [x] Pull request is made against `development` branch.
- [x] My code follows the style guidelines of this project.
- [x] I run `npm run format` before creating the pull request.
- [x] I have performed a self-review of my own code.
  
- [x] My UI is responsive
- [x] UI made replicates the design provided

### CODE REVIEW

**Comment by nishant-rajput** [comment]
@Abhishek-kumar09  I mistakenly converted my draft PR to a ready-for-review PR

**Comment by Abhishek-kumar09** [comment]
I have done some changes here, please give a look.

File: src/Routes.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import React, { Suspense } from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LoadingScreen from 'src/components/LoadingScreen';
 import Navigation from 'src/components/dashboard';
+import ProfileInfo from 'src/components/ProfileInfo';
```
Comment (Abhishek-kumar09) [inline src/Routes.js]: We don't need to create a new route, we already have a route for profile.

**Comment by nishant-rajput** [comment]
@Abhishek-kumar09 saw the changes, thanks for the help. Also, I added the component to `/profile`.

**Pull request merged**